gateway/gateway.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/textileio/powergate/reputation"
 )
 
-const numTopMiners = 30
+const numTopMiners = 100
```

Comment:
Bump it a bit more, so arbitrary. Eventually, we can support (bounded) query-parameters, etc.